README_VERITABLE.md

```diff
@@ -41,7 +41,7 @@ The project aims to enable supply chains to share insights and data across multi
 
 ## Quick start
 
-The REST API provides an OpenAPI schema that can easily be viewed using the SwaggerUI that is provided with the server. The docs can be viewed on the `/docs` endpoint (e.g. http://localhost:3000/docs).
+The REST API provides an OpenAPI schema that can easily be viewed using the SwaggerUI that is provided with the server. The docs can be viewed on the `/api-docs` endpoint (e.g. http://localhost:3000/api-docs).
```

Comment:
kind of incorrect. The SwaggerUI is viewed at `/swagger` but the OpenAPI spec is server at `/api-docs`